include/posix/unistd.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __POSIX_UNISTD_H__
+#define __POSIX_UNISTD_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include_next <unistd.h>
+#include "sys/types.h"
+
+/**
+ * @brief suspend execuetion for specified seconds.
```

Comment:
Typo: execuetion

I'm not a native speaker, but shouldn't this be `suspend execution of current thread for the specified number of seconds`?